WordPress/Classes/Models/ManagedAccountSettings+CoreDataProperties.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+import CoreData
+
+extension ManagedAccountSettings {
```

Comment:
I like the separation. I just think the contents of the class/extension should be swapped. My main reason is that none of the code currently in the class would work if you take away the extension, but not the other way around